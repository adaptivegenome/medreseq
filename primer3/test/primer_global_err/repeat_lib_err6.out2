../src/primer3_core: Illegal weight in mispriming library repeat_lib6
