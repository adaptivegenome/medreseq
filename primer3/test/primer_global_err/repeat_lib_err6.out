SEQUENCE_ID=seq1
SEQUENCE_TEMPLATE=ATGCTAGCTAGTCGATCGATGCTAGCTAGTCGATCTACTATCATCATGCTAGCATGGGGCATCTGTGGGATGCTGTCGTACTGATCGATGCTAGCTAGCTAGTCGATCGTAGCTATCATCATCTAGTCATCGTAGCTATCATCGTAGC
PRIMER_MISPRIMING_LIBRARY=repeat_lib6
PRIMER_ERROR=Illegal weight in mispriming library repeat_lib6
=
